### DISCUSSION TITLE
How to use the touch screen to scroll in vim/less/etc?

### DISCUSSION THREAD

**Discussion by UrsaDK** [ish-app/ish#2112] [Q&A]
Whenever I try to scroll using the screen in Vim or less, I often end up scrolling through the terminal's scrollback buffer instead. Most of the time, that's not what I want, as I'm just trying to scroll through the file I'm viewing.

Obviously, there are other ways to scroll through that file (`Ctrl-U`, `Ctrl-E`, `zb`, etc) but sometimes, especially when I'm reading, I just want to scroll down without bothering with the keyboard. 🙂

So my question is, has anyone figured out how to scroll in vim or less using the screen?

**Comment by elgithubo** [comment]
Sorry for squading this, but how do you send ctrl?

**Reply by saagarjha** [reply]
Hit the key that looks like this: ⌃

**Reply by elgithubo** [reply]
says "command not understood" in "top".

**Comment by Yousousen** [comment]
I'm looking for a way to do this as well. As an alternative, one can map `<up>` and `<down>` to `<c-u>` and `<c-d>` respectively. What is great about these mappings is that `<up>` and `<down>` can be held down with the button on the extra keys bar.
